www/calendar/inner.js

```diff
@@ -868,7 +885,7 @@ define([
                     });
                 });
                 if (APP.loggedIn) {
-                    APP.$calendars.append(h('div.cp-calendar-entry.cp-ghost', importTemp));
+                    $tempCalendarEntries.append(h('div.cp-calendar-entry.cp-ghost', importTemp));
```

Comment:
The `$tempCalendarEntries` container is hidden by default so neither the calendar entry with its title and actions nor the "import" button are visible. It seems the goal was to display the "show calendars" button here but, in my opinion, it doesn't make sense to show this button when there is a single entry.